src/components/common/ColonyMembers/ColonyMembers.tsx

```diff
@@ -69,7 +79,7 @@ const ColonyMembers = () => {
           <TotalReputation selectedDomainId={selectedDomainId} />
           <MemberControls isRootDomain={isRootDomain} />
           <MembersFilter
-            handleFiltersCallback={setFilters}
+            handleFiltersCallback={checkFilters}
```

Comment:
The convention seems to be to name those `onSomeEvent` instead

Reply:
@jakubcolony in this case the function which is now called `handleFilterChange` is passed to MembersFilter -> Filter -> Select and only once it reaches the Select is the function passed to onChange. So does it make sense to name the props that pass the function the same as the function `handleFilterChange` or change to the `onSomeEvent' pattern?

Reply:
I would still say so. I like the idea that `handleX` is what you define inside a particular component while `onX` is callback passed from somewhere else. It seem to be a very common pattern